utils/atool/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2006-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=atool
+PKG_VERSION:=0.39.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://savannah.nongnu.org/download/atool/
```

Comment:
PKG_SOURCE_URL:=@SAVANNAH/$PKG_NAME

Use the available URL variables (which also provides mirrors)